src/SDL_internal.h

```diff
@@ -133,6 +133,16 @@
 #include <float.h>
 #endif
 
+#ifdef HAVE_LIBC
+#define SDL_abort() abort()
+#else
+#define SDL_abort() do {                                        \
```

Comment:
We probably want to declare `SDL_ExitProcess` outside the block scope. If I remember right, some compilers don't like that. Do we need the `__WATCOMC__` stuff here?

Reply:
(Sorry for late reply - noticed this one late.)

> Can Watcom compile SDL3 for Windows/Linux? @sezero

Never tested building sdl3 itself using watcom, and never will: unsupported. So, watcom stuff can be removed from private sdl3 sources.

P.S.: Headers are still compatible, though: sdl2-compat can still build for windows using watcom.